src/scripts/generate_code_snippets.mjs

```diff
@@ -0,0 +1,91 @@
+import fs from "fs/promises";
```

Comment:
note: here's the script we use for the code translations

Reply:
this is a js module file `.mjs` [what-is-the-difference-between-js-and-mjs-files](https://stackoverflow.com/questions/57492546/what-is-the-difference-between-js-and-mjs-files)?